### PR TITLE
Propagate the noise field.

### PR DESCRIPTION
**Pull request by alimhassam**
Propagate the noise field of glucose data so it can be used by determine_basal.

### CODE REVIEW

**Comment by alimhassam** [comment]
without this, the new noise logic doesn't work, see comments here: https://github.com/openaps/oref0/pull/735#issuecomment-347035387

File: lib/glucose-get-last.js [hunk 1]
```diff
@@ -65,6 +65,7 @@ var getLastGlucose = function (data) {
     return {
         delta: Math.round( last_delta * 100 ) / 100
         , glucose: Math.round( now.glucose * 100 ) / 100
+        , noise: now.noise
```
Comment (scottleibrand) [inline lib/glucose-get-last.js]: Should we round it to the appropriate precision to make sure we don't have floating point math fun like 3.00000000000001?

Comment (alimhassam) [reply lib/glucose-get-last.js]: i guess we could, probably wouldn't hurt, but at the moment, i only saw integer values for this field, but let me fix that.

Comment (alimhassam) [reply lib/glucose-get-last.js]: fixed

**Pull request merged**